app/assets/javascripts/jquery.extentions.js

```diff
@@ -22,3 +22,7 @@ jQuery.humanize = function(str) {
             return match.charAt(0).toUpperCase() + match.slice(1);
         });
 };
+
+jQuery.escapeHTML = function(str) {
+  return $("<div></div>").text(str).html();
+};
```

Comment:
We could consider using [`_.escape()`](http://underscorejs.org/#escape) if we wanted to bring in underscore.js (really useful for other utility type things as well), then we don't have to maintain this.

At the very least we should use one of the suggestions [here](http://stackoverflow.com/a/6020820).  Note the first comment on that answer.